resources/testharness.js

```diff
@@ -3168,18 +3169,33 @@
         );
     };
 
-    Tests.prototype.fetch_tests_from_worker = function(worker) {
+    Tests.prototype.fetch_tests_from_worker = function(promiseOrWorker) {
```

Comment:
So I this this is a good change, but technically this kind of API change should have an RFC. Maybe if others are happy with this change we could skip that in this case, but I'd like feedback from at least some of @Ms2ger, @foolip, and @gsnedders 

It definitely requires a docs update in `docs/writing_tests/testharness-api.md`.

Reply:
I was wondering whether to update the "Determining when all tests are complete" section, but since that section seems to skip a number of the conditions in `Tests.prototype.all_done`, it seems like this change wouldn't be significant enough to mention there.

Reply:
I don't have enough of this code paged in to be confident in the change, but I won't object to it.

Reply:
This is now moot; see https://github.com/web-platform-tests/wpt/pull/29970#issuecomment-916781371